app/components/public/stream/join-video.ts

```diff
@@ -36,11 +36,15 @@ export default class JoinVideo extends Component<Args> {
 
   @action
   openPanel(): void {
+    let currentRoute  = this.router.currentRoute.name;
+    if (currentRoute === 'public.session.view' || currentRoute === 'public.exhibition.view') {
+      currentRoute = 'public';
```

Comment:
Why?

Reply:
on the session view page it needs session-Id also to transition.

Reply:
There must be a better solution